### DISCUSSION TITLE
Microsoft Learn for GitHub

### DISCUSSION THREAD

**Discussion by RichCoastCR** [community/community#146704] [GitHub Education] [answered]
### Hi everyone,

I am looking for a course from Microsoft about master essential GitHub skills

**Comment by arcesoftware** [comment, marked as answer]
I think this is what you are looking for: 

https://learn.microsoft.com/en-us/training/github/

**Reply by RichCoastCR** [reply]
Perfect this is what I needed!

**Comment by RichCoastCR** [comment]
The link about is very useful, check it out.
